editable-list.tsx

```diff
@@ -0,0 +1,95 @@
+/**
```

Comment:
Did you mean to add files at the root of the repo?

Reply:
errare humanum est. I amended the commit and moved the files to their correct places. sorry!